install_files/udev/razer_mount

```diff
@@ -42,15 +42,17 @@ if [ ! -d /sys/bus/hid/drivers/"$DRIVER" ] ; then
 	mesg "Modprobed $DRIVER"
 fi
 
-if [ -d /sys/bus/hid/drivers/hid-generic/"$DEVICE_ID" ] ; then
-	# Unbind from hid
-	mesg "Unbinding $DEVICE_ID from hid-generic"
-	printf '%s' "$DEVICE_ID" > /sys/bus/hid/drivers/hid-generic/unbind
-	mesg "Binding $DEVICE_ID to $DRIVER"
-	printf '%s' "$DEVICE_ID" > /sys/bus/hid/drivers/"$DRIVER"/bind
-	sleep 0.1
-	mesg "Finished binding $DEVICE_ID"
-fi
+for GENERIC_DRIVER in "razer" "hid-generic"; do
+	if [ -d /sys/bus/hid/drivers/"$GENERIC_DRIVER"/"$DEVICE_ID" ] ; then
+		# Unbind from hid
+		mesg "Unbinding $DEVICE_ID from $GENERIC_DRIVER"
+		printf '%s' "$DEVICE_ID" > /sys/bus/hid/drivers/"$GENERIC_DRIVER"/unbind
+		mesg "Binding $DEVICE_ID to $DRIVER"
+		printf '%s' "$DEVICE_ID" > /sys/bus/hid/drivers/"$DRIVER"/bind
```

Comment:
I'd put the bind part outside the loop as we logically don't want to bind twice. Maybe if it doesn't find any driver to unbind from then exit?
Or not, I'll look at it later again

Reply:
Unless you're seeing something I don't, the logic seems correct with this loop:

**- BlackWidow Classic** (affected)

* Is device binded to `razer`?
  * Yes, unbind `razer` 
  * Bind `razerkbd`
* Is device binded to `hid-generic`?
  * No

**- BlackWidow Chroma**

* Is device binded to `razer`?
  * No
* Is device binded to `hid-generic`?
  * Yes, unbind `hid-generic`
  * Bind `razerkbd`

**- Random device**

* Is device binded to `razer`?
  * No
* Is device binded to `hid-generic`?
  * No

Reply:
Was just nitpicking but it really doesn't matter